sklearn/ensemble/forest.py

```diff
@@ -922,6 +922,26 @@ class labels (multi-output problem).
         was never left out during the bootstrap. In this case,
         `oob_decision_function_` might contain NaN.
 
+    Examples
+    --------
+    >>> from sklearn.ensemble import RandomForestClassifier
+    >>> from sklearn.datasets import make_classification
+    >>>
+    >>> X, y = make_classification(n_features=4, n_informative=2,
+    ...                            n_redundant=0, random_state=0)
+    >>> clf = RandomForestClassifier(max_depth=5, random_state=0)
+    >>> clf.fit(X, y)
+    RandomForestClassifier(bootstrap=True, class_weight=None, criterion='gini',
+                max_depth=5, max_features='auto', max_leaf_nodes=None,
+                min_impurity_decrease=0.0, min_impurity_split=None,
+                min_samples_leaf=1, min_samples_split=2,
+                min_weight_fraction_leaf=0.0, n_estimators=10, n_jobs=1,
+                oob_score=False, random_state=0, verbose=0, warm_start=False)
+    >>> print(clf.feature_importances_)
```

Comment:
I mean i don't want to overcomplicate this, but which are the important features? I think we can either get that information out of the make_classification or no shuffle the features so we know? And then maybe set something in the RF so that it's actually visible?